org.sonarlint.eclipse.ui/plugin.xml

```diff
@@ -1099,6 +1120,10 @@
           class="org.sonarlint.eclipse.ui.internal.command.ShowRuleDescriptionCommand"
           commandId="org.sonarlint.eclipse.ui.internal.command.ShowRuleDescriptionCommand">
     </handler>
+    <handler
+          class="org.sonarlint.eclipse.ui.internal.command.HelpOnIconsCommand"
+          commandId="org.sonarlint.eclipse.ui.internal.command.HelpOnIconsCommand">
+    </handler>
```

Comment:
Should we make them visible only when necessary, e.g. when clicking on a marker? What I did on the [other PR](https://github.com/SonarSource/sonarlint-eclipse/pull/520) might help